torchgeo/datamodules/__init__.py

```diff
@@ -46,6 +47,7 @@
     "GID15DataModule",
     "InriaAerialImageLabelingDataModule",
     "LandCoverAIDataModule",
+    "L8BiomeDataModule",
```

Comment:
Technically L8 sorts higher than LandCover or Landsat, so we should reorder these in all files.